types/symlink.sh

```diff
@@ -8,16 +8,16 @@ case "$action" in
     echo "> symlink .vimrc ~/code/dotfiles/configs/vimrc"
     ;;
   status)
-    if bake [ -h $target ]; then
-      existing_source=$(bake readlink $target)
-      if [ "$existing_source" != $source ]; then
+    if bake [ -h "\"$target\"" ]; then
+      existing_source=$(bake readlink \"$target\")
+      if [ "$existing_source" != "$source" ]; then
         echo "received source for existing symlink: $existing_source"
         echo "expected source for symlink: $source"
-        return $STATUS_MISMATCH_CLOBBER
+        return $STATUS_MISSING
       else
```

Comment:
MISMATCH_CLOBBER is really the intention here – it lets the runner know that the target exists, but not in the desired form, and overwriting it can (theoretically) proceed without data loss. The runner then gives the user the option to do that overwriting or not.